code/modules/projectiles/updated_projectiles/gun_system.dm

```diff
@@ -946,6 +946,12 @@ and you're good to go.
 			projectile_to_fire.def_zone = user.zone_selected
 			if(C.stagger)
 				gun_scatter += 30
+		
+		var/obj/item/clothing/head/head_slot = user.get_item_by_slot(SLOT_HEAD)
+		if(istype(head_slot))
+			gun_accuracy_mult += head_slot.accuracy_mod
+		else
+			gun_accuracy_mult += 0.20 //nothing on your head? well if you're gonna die then might as well be accurate.
```

Comment:
```suggestion
			gun_accuracy_mult += 0.2 //nothing on your head? well if you're gonna die then might as well be accurate.
```

Reply:
Though this shouldn't be checked every time someone shoots. This should be a mob variable that gets changed on equip/unequip.